src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -34,35 +29,24 @@ class ThirdPartyAIService extends Component {
       this.serviceRequestCompleteHandler
     );
     const { serviceSpecJSON, protoSpec } = await this.fetchServiceSpec(org_id, service_id);
-    this.fetchAIServiceComponent();
-    this.setState({ loading: false, serviceSpecJSON, protoSpec });
+    this.serviceSpecJSON = serviceSpecJSON;
+    this.protoSpec = protoSpec;
+    await this.fetchUserFeedback();
```

Comment:
Need not to await this call.. Let it happen asynchronously

Reply:
rest LGTM

Reply:
Addressing this in the following PR which fixes the sandbox mode.